masonry/src/widgets/text_input.rs

```diff
@@ -150,6 +194,21 @@ impl Widget for TextInput {
         let pos = padding.place_down(pos);
         ctx.place_child(&mut self.text, pos);
 
+        let text_is_empty = {
+            ctx.get_raw(&mut self.text)
+                .0
+                .text()
+                .chars()
+                .next()
+                .is_none()
+        };
```

Comment:
It seems strange to exclude the IME contents from the detection for whether this is empty.

<img width="269" height="54" alt="Image" src="https://github.com/user-attachments/assets/e1e8dfa0-eacb-4424-b198-50ddc3d29ba0" />

Reply:
So how do I properly detect an empty string?